api/src/org/apache/cloudstack/api/command/admin/offering/CreateDiskOfferingCmd.java

```diff
@@ -130,6 +130,9 @@
             description = "Hypervisor snapshot reserve space as a percent of a volume (for managed storage using Xen or VMware)")
     private Integer hypervisorSnapshotReserve;
 
+    @Parameter(name = ApiConstants.CACHE_MODE, type = CommandType.STRING, required = true ,description = "the cache mode to use for this disk offering. none, writeback or writethrough")
+    private String cacheMode;
+
 /////////////////////////////////////////////////////
```

Comment:
could this be an enum type?

Reply:
Done. I changed the cache mode to an enum.